src/seedu/addressbook/data/AddressBook.java

```diff
@@ -54,6 +54,25 @@ public void removePerson(ReadOnlyPerson toRemove) throws PersonNotFoundException
         allPersons.remove(toRemove);
     }
 
+    public void removePerson(Person toRemove) throws PersonNotFoundException {
+        allPersons.remove(toRemove);
+    }
+    /**
+     * Edit the equivalent person from the address book.
+     *
+     * @throws PersonNotFoundException if no such Person could be found.
+     */
+    public void editPerson(ReadOnlyPerson personOut, Person personIn) throws Exception {
```

Comment:
No JUnit test for this method in AddressBookTest

Reply:
To be fair, I only overloaded their editPerson method, and their method did not have testing either